content/v2.0/query-data/flux/first-last.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Query first and last values
+seotitle: Query first and last values in Flux
+list_title: First and last
+description: >
+  Use the [`first()`](/v2.0/reference/flux/stdlib/built-in/transformations/selectors/first/) or
+  [`last()`](/v2.0/reference/flux/stdlib/built-in/transformations/selectors/first/) functions
+  to return the first or last point in an input table.
+weight: 210
+menu:
+  v2_0:
+    parent: Query with Flux
+    name: First & last
+v2.0/tags: [query]
+list_code_example: |
+  ```js
+  data
+    |> first()
+
+  // OR
+
+  data
+    |> last()
+  ```
+---
+
+Use the [`first()`](/v2.0/reference/flux/stdlib/built-in/transformations/selectors/first/) or
+[`last()`](/v2.0/reference/flux/stdlib/built-in/transformations/selectors/first/) functions
+to return the first or last point in an input table.
+
+```js
+data
+  |> first()
+
+// OR
+
+data
+  |> last()
+```
+
+{{% note %}}
+By default, InfluxDB returns results sorted by time, however you can use the
+[`sort()` function](/v2.0/reference/flux/stdlib/built-in/transformations/sort/)
+to change how results are sorted.
+**`first()` and `last()` respect the sorting of input tables.**
```

Comment:
does "respect the sorting of...." mean these functions return results sorted by time by default, or specified sort order with sort function?

Reply:
Updated a little bit to clarify.